### ISSUE TITLE
ER: Update text on Team Meeting page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#6162]
### Emergent Requirement - Problem
- We need to update the team meeting page to reflect that people can attend any team meeting where that team also has open roles

### Issue you discovered this emergent requirement in
- #4699 

### Date discovered 
2024-01-25

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@ExperimentsInHonesty 

### What happens if this is not addressed
people will go to team meetings where the teams don't need people, which is a burden on team leads.

### Resources
https://www.hackforla.org/project-meetings

#### Current text on https://www.hackforla.org/project-meetings
>Please review the listing of project team meeting times to find a project that fits your schedule. You are welcome to attend a project team meeting after you have completed [Onboarding](https://www.hackforla.org/getting-started). Details about a team's meeting can be obtained in each team's slack channel.

#### Current text on https://www.hackforla.org/events/
>Please review the listing of project team meeting times to find a project that fits your schedule. You are welcome to attend a project team meeting after you have completed [Onboarding](https://www.hackforla.org/getting-started).

#### Replacement text 
> We have teams that meet at various times. You are welcome to attend a project team meeting after you have completed Onboarding, and if the team currently has an open role. You can find open roles at the end of the [Onboarding](https://www.hackforla.org/getting-started).

Here is the replacement text in an easy to copy format:
```
We have teams that meet at various times. You are welcome to attend a project team meeting after you have completed Onboarding, and if the team currently has an open role. You can find open roles at the end of the <a href="https://www.hackforla.org/getting-started" target="_blank">Onboarding</a>.
```

### Recommended Action Items
- [x] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
We discussed this at the Dev/PM meeting on March 11th and had some questions.

- 
  > We need to properly capitalize Slack where appropriate on the site
  
  There is only one area on the site where Slack isn't capitalized and it is getting replaced with the replacement text. Did you want to check the site only or the whole codebase, such as issue templates and the CONTRIBUTING.md file too?



-
  > We have teams that meet at various times.  You are welcome to attend a project team meeting after you have completed if the team currently has an open role.  Find open roles at the end of [Onboarding](https://www.hackforla.org/getting-started). 
  
  To make it clearer, we suggest this edit:
  > We have teams that meet at various times. You are welcome to attend a project team meeting after you have completed [Onboarding](https://www.hackforla.org/getting-started) and if the team currently has an open role. You can find open roles at the end of the onboarding session.
  
  What do you think?

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng 

- #6163 is the ER to replace slack with Slack everywhere else on the site.
- Change language to 
```
We have teams that meet at various times. You are welcome to attend a project team meeting after you have completed Onboarding, and if the team currently has an open role. You can find open roles at the end of the [Onboarding](https://www.hackforla.org/getting-started).
```
>We have teams that meet at various times. You are welcome to attend a project team meeting after you have completed Onboarding, and if the team currently has an open role. You can find open roles at the end of the [Onboarding](https://www.hackforla.org/getting-started).

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 

- > [ER: Find all the places that slack should be Slack and fix #6163](https://github.com/hackforla/website/issues/6163) is the ER to replace slack with Slack everywhere else on the site.
  
  Ok. Then, for this ER, under _Emergent Requirement - Problem_, I have removed:
  > - We need to properly capitalize Slack where appropriate on the site
  
  because the Slack capitalization is addressed in the other ER (#6163).


- > Change language to...
  > > We have teams that meet at various times. You are welcome to attend a project team meeting after you have completed Onboarding, and if the team currently has an open role. You can find open roles at the end of the [Onboarding](https://www.hackforla.org/getting-started).
  
  Under _Replacement text_, I have edited the replacement text to the updated one.


- This ER is ready for prioritization

**Comment by A-Wu5** [comment]
1. W-F 7PM - 8PM
2. Today

**Comment by JessicaLucindaCheng** [comment]
@A-Wu5 Thanks for taking up this ER! 

- I made a change to the text under "Here is the replacement text in an easy to copy format:" (see above); the text now uses an `a` HTML tag for the Onboarding link instead of Markdown formatting so that it will be displayed on the website properly as a link. 

- Also, once you have written the issue, please leave a comment in this ER with a link to the issue, so we know which issue is for this ER.

**Comment by A-Wu5** [comment]
- The issue linked is #6539 and I've updated the text to replace with.

**Comment by A-Wu5** [comment]
1. Waiting on dev lead
2. N/A

**Comment by jphamtv** [comment]
- Issue #6539 reviewed and ready for prioritization
